app/assets/javascripts/dashboard.js

```diff
@@ -130,28 +77,111 @@
 
   getLocalStorageActivity();
 
-
   // allow native stylesheets to determine visibility on 
   $(window).on('resize', function() {
-
     $('.activity .col-md-6').css('display', '');
     getLocalStorageActivity();
-
   });
 
 
   $('.activity-dropdown .dropdown-toggle').click(function(e) {
```

Comment:
OK, this is now empty - shall we remove it?

Reply:
What do you mean? What should be removed?